src/Sarif/HashUtilities.cs

```diff
@@ -7,11 +7,35 @@
 
 namespace Microsoft.CodeAnalysis.Sarif
 {
+    public class HashData
+    {
+        string md5;
```

Comment:
Conventions:

* We explicitly specify the accessibility of each member, even if it is the default.
* We prefix instance field names with "`_`".

So:

```
    private string _md5;
```

But, our